src/components/Popover/tests/Popover.test.tsx

```diff
@@ -95,6 +95,21 @@ describe('<Popover />', () => {
     expect(popoverOverlay.prop('preferredAlignment')).toBe('left');
   });
 
+  it("passes 'preferInputActivator' to PopoverOverlay", () => {
+    const popover = mountWithAppProvider(
```

Comment:
small nit, we're trying to switch over the react-testing as much as possible. If you can, change this to use `mountWithApp` instead and avoid the testId.

Reply:
We actually thought about that, but wanted to keep the test file consistent and not use different testing libraries in one file.
Depending on the effort, we'd try to refactor the entire file.

Reply:
That would be great if you could. We have been mixing though :|

Reply:
We updated the entire test file, could you please take a look?

Reply:
That's awesome thank you! 1 test is failing and there's a merge conflict but apart from that, it looks good!